### PR TITLE
Fix wrong namespaces and imports

### PR DESCRIPTION
**Pull request by imanghafoori1**
Hello everyone, I checked this application with laravel-microscope package to find the wrong namespaces and imports and it found a couple of issues. 
The run commands are:
- php artisan check:psr4
- php artisan check:import

Hope it helps.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
         "willvincent/laravel-rateable": "^2.0"
     },
     "require-dev": {
+        "imanghafoori/laravel-microscope": "^1.0",
```
Comment (mattstauffer) [inline composer.json:42]: I'd be happy to merge this PR, but we don't need to add this package permanently. Thanks!

**Comment by imanghafoori1** [comment]
I deleted the second commit.

**Comment by imanghafoori1** [comment]
I think you you may have intended to name the /search route as your tests expect a 'search' route to exist.

image

**Comment by imanghafoori1** [comment]
this mistake was also revealed when scanning by microscope. ;)
which relates to the second commit.

image

**Comment by mattstauffer** [comment]
Thanks @imanghafoori1!

**Comment by imanghafoori1** [comment]
@mattstauffer Your are welcome man.

**Pull request merged**